### PR TITLE
DPC-1027: Add Public Key operations to CLI

### PR DESCRIPTION
**Pull request by nickrobison-usds**
**Why**

As part of manually on-boarding organizations, we need any easy way to manage public keys. We had some of the features already, but some were missing and needed to be added to the CLI.

**What Changed**

Added missing admin tasks for public keys. These functions are only accessible when connected to the AWS VPC via the VPN. We can upload, list and delete keys via the admin tasks and the CLI commands.

**Choices Made**

The changes were pretty straightforward, there are also some scattered changes as I added unit tests for our existing admin tasks, this required some small changes to existing files to expose constructors and such.

**Tickets closed**:

DPC-1027

**Future Work**

**Checklist**

- [x] All tests are passing via `make ci-app` (app change) and `make ci-web` (website change)
- [x] Swagger documentation has been updated
- [x] FHIR documentation has been updated
- [x] Any required dpc-ops changes have a PR submitted and mentioned in this ticket
- [x] Any manual migration steps are documented, scripts written (where applicable), and tested
- [x] Before merging, any required dpc-ops changes have been approved and merged into master of the dpc-ops repo

### CODE REVIEW

**Comment by DeirdreHolub** [comment]
I am unable to run the command. I have tried:

File: dpc-api/src/main/java/gov/cms/dpc/api/cli/keys/KeyCommand.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package gov.cms.dpc.api.cli.keys;
+
+import gov.cms.dpc.api.cli.AbstractCommandTree;
+
+public class KeyCommand extends AbstractCommandTree {
+
+    protected KeyCommand() {
+        super("key", "Public key related commands");
+    }
```
Comment (DeirdreHolub) [inline dpc-api/src/main/java/gov/cms/dpc/api/cli/keys/KeyCommand.java]: Register subcommands here. See details in other comment.

Comment (nickrobison-usds) [reply dpc-api/src/main/java/gov/cms/dpc/api/cli/keys/KeyCommand.java]: Well that was embarrassing, thanks for catching it!

File: dpc-api/src/main/java/gov/cms/dpc/api/cli/keys/KeyUpload.java [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+        super("upload", "Upload organization public key");
+    }
+
+    @Override
+    public void addAdditionalOptions(Subparser subparser) {
+        subparser
+                .addArgument("id")
+                .required(true)
+                .dest("org-reference")
+                .help("ID of Organization to list tokens");
```
Comment (embh) [inline dpc-api/src/main/java/gov/cms/dpc/api/cli/keys/KeyUpload.java]: Should this say something more like, `ID of organization to create token for`?

File: dpc-api/src/main/java/gov/cms/dpc/api/cli/keys/KeyUpload.java [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+        super("upload", "Upload organization public key");
+    }
+
+    @Override
+    public void addAdditionalOptions(Subparser subparser) {
+        subparser
+                .addArgument("id")
+                .required(true)
+                .dest("org-reference")
+                .help("ID of Organization to upload keys tokens");
```
Comment (embh) [inline dpc-api/src/main/java/gov/cms/dpc/api/cli/keys/KeyUpload.java]:
```suggestion
                .help("ID of Organization to upload key");
```

**Comment by DeirdreHolub** [comment]
For documentation purposes:

deirdreholub 12:19 dpc-app (nickrobison/DPC-1027-key-cli) $ java -jar dpc-api/target/dpc-api.jar key upload -l djKey --host http://localhost:9903/tasks 46ac7ad6-7487-4dd0-baa0-6e2c8cae76a0 /Users/deirdreholub/code/go/src/dpcclient/djSandbox-public.pem 
WARN  [2020-02-19 12:21:02,630] com.squarespace.jersey2.guice.JerseyGuiceUtils: It appears jersey2-guice-spi is either not present or in conflict with some other Jar: ServiceLocatorGeneratorImpl(hk2-locator, 1016856028)
Connecting to API service at: http://localhost:9903/tasks
Organization token: {"id":"6cb85a8a-7864-4b48-935b-fcaeda17c4bd","publicKey":"-----BEGIN PUBLIC KEY-----deleted for brevity-----END PUBLIC KEY-----\n","createdAt":"2020-02-19T19:21:03.720+00:00","label":"djKey"}
deirdreholub 12:21 dpc-app (nickrobison/DPC-1027-key-cli) $

**Pull request merged**